### ISSUE TITLE
Cannot reproduce NeRF results on LLFF Fern dataset

### ISSUE THREAD

**Issue by sergeyprokudin** [facebookresearch/pytorch3d#1292]
Hi Pytorch3d team,

I was recently trying to reproduce the results of [Pytroch3d NeRF implementation](https://github.com/facebookresearch/pytorch3d/tree/main/projects/nerf) on the LLFF "fern" dataset. No changes on my side were made to the code, I've just run the provided command: 

`python ./train_nerf.py --config-name fern`

However, the results I'm getting on the validation set are significantly lower than the ones reported in the readme (27.9 PSNR vs 17.8 I'm getting). I somehow feel this might be connected to the behavior described in the [following open issue](https://github.com/facebookresearch/pytorch3d/issues/868).

It would be great if you can look into this. Having reproducible NeRF baselines in a modular and extendable framework like Pytorch3d would be of great help for future research on neural rendering. 

Thanks in advance,

~Sergey

**Comment by gkioxari** [comment]
It's my understanding that we are deprecating `projects/nerf` with the new implicitron library

**Comment by sergeyprokudin** [comment]
I see, thanks for the clarification! It would be great to have the code for reproducing NeRF LLFF and Lego within the implicitron framework then.

**Comment by gkioxari** [comment]
@bottler @davnov134 Can you point Sergey to the code that reproduces NeRF for LLFF and Lego?

**Comment by sergeyprokudin** [comment]
Hi folks! Any update on this?

**Comment by weizhenFrank** [comment]
@sergeyprokudin Bro, I also have the same problem as you. I run the code 
```python ./train_nerf.py --config-name fern``` and ```python ./test_nerf.py --config-name fern```. But I didn't fully reproduce the result. My result is as the screenshot shows.
<img width="1491" alt="image" src="https://user-images.githubusercontent.com/37463904/207943561-e72f48d7-e867-45d5-9c08-6e607f394560.png">
However, the repo here said the PSNR should be 
```+----------------+------------------+------------------+-----------------+
| Implementation |  Lego: test PSNR |  Fern: test PSNR |  training speed |
+----------------+------------------+------------------+-----------------+
| TF (official)  |             31.0 |             27.5 |  0.24 sec/it    |
| PyTorch3D      |             32.7 |             27.9 |  0.18 sec/it    |
+----------------+------------------+------------------+-----------------+```

**Comment by weizhenFrank** [comment]
Update: Just as https://github.com/facebookresearch/pytorch3d/issues/868 said, this implementation has some problems for non-square images. But I successfully reproduce the lego result.
